packages/core/conductor/src/nodes/function.ts

```diff
@@ -33,10 +33,7 @@ export const executeFunction = (
   return Effect.gen(function* () {
     const functionCallService = yield* RemoteFunctionExecutionService;
 
-    const result = yield* Effect.tryPromise({
-      try: () => functionCallService.callFunction(path, input),
-      catch: (e) => e,
-    });
+    const result = yield* functionCallService.callFunction(path, input).pipe(Effect.catchAll((e) => Effect.succeed(e)));
 
```

Comment:
what is `path`?

Reply:
`/${functionId}`

Reply:
Do not know why it is prefixed with `/` though